docs/guides/taxes.rst

```diff
@@ -49,3 +53,28 @@ Fetching taxes
 
     By default, Saleor is making requests to the VatLayer API through HTTP (insecure),
     if you are using a paid VatLayer subscription, you may want to set the settings ``VATLAYER_USE_HTTPS`` to ``True``.
+
+
+Avalara
+-------
+
+All configuration and account setup should be done on the Avalara's admin panel. Saleor uses standard Avalata's  API to fetch tax codes, calculate taxes during the checkout process and transfer data directly to the Avalara's system.
+
+Configuration
+^^^^^^^^^^^^^
+
+Only basic data needs to be provided on Saleor side. You need to fill up variables:
+``AVATAX_USERNAME_OR_ACCOUNT``, ``AVATAX_PASSWORD_OR_LICENSE``, ``AVATAX_USE_SANDBOX``, ``AVATAX_COMPANY_NAME``, ``AVATAX_AUTOCOMMIT``, (:ref:`see description of variables.<tax_environment_variables>`)
+
+
+Avalara requires a company address to properly calculate taxes.
+The address must be assigned in the dashboard's general section.
```

Comment:
```suggestion
The address must be assigned in the general section of Avalara's dashboard.
```

Reply:
It is in our dashboard not in Avalara's admin panel.